src/WrapStory.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components'
+
+export default class WrapStyledComponentsThemePicker extends React.Component {
```

Comment:
how about we call it like the file name? `class WrapStory extends React.Component` ?